### PR TITLE
Fix for missing swift frameworks on iOS <12.2

### PR DESCRIPTION
**Pull request by JCash**

### CODE REVIEW

File: facebook/ext.manifest [hunk 1]
```diff
@@ -10,23 +10,23 @@ platforms:
             frameworks: ['Security', 'Accelerate']
             weakFrameworks: ['Accounts', 'Social', 'AdSupport']
             flags: ['-fcxx-modules', '-fmodules']
-            linkFlags:  ['-ObjC', '-Wl,-rpath,/usr/lib/swift']
+            linkFlags:  ['-ObjC', '-Wl,-rpath,/usr/lib/swift', '-Wl,-rpath,@executable_path/Frameworks']
```
Comment (JCash) [inline facebook/ext.manifest:13]: This adds another runtime search path for iOS<12.2

File: facebook/libhelper/ios/update.sh [hunk 2]
```diff
@@ -15,3 +15,28 @@ cp -v -r $TMP/FBSDKShareKit.framework ../../lib/ios/
...
+${SDK}/usr/bin/bitcode_strip ${SDK}/usr/lib/swift-5.0/iphoneos/libswiftDispatch.dylib -r -o ${DST}/libswiftDispatch.dylib
+${SDK}/usr/bin/bitcode_strip ${SDK}/usr/lib/swift-5.0/iphoneos/libswiftFoundation.dylib -r -o ${DST}/libswiftFoundation.dylib
+${SDK}/usr/bin/bitcode_strip ${SDK}/usr/lib/swift-5.0/iphoneos/libswiftMetal.dylib -r -o ${DST}/libswiftMetal.dylib
+${SDK}/usr/bin/bitcode_strip ${SDK}/usr/lib/swift-5.0/iphoneos/libswiftObjectiveC.dylib -r -o ${DST}/libswiftObjectiveC.dylib
+${SDK}/usr/bin/bitcode_strip ${SDK}/usr/lib/swift-5.0/iphoneos/libswiftPhotos.dylib -r -o ${DST}/libswiftPhotos.dylib
+${SDK}/usr/bin/bitcode_strip ${SDK}/usr/lib/swift-5.0/iphoneos/libswiftQuartzCore.dylib -r -o ${DST}/libswiftQuartzCore.dylib
+${SDK}/usr/bin/bitcode_strip ${SDK}/usr/lib/swift-5.0/iphoneos/libswiftUIKit.dylib -r -o ${DST}/libswiftUIKit.dylib
+${SDK}/usr/bin/bitcode_strip ${SDK}/usr/lib/swift-5.0/iphoneos/libswiftos.dylib -r -o ${DST}/libswiftos.dylib
+${SDK}/usr/bin/bitcode_strip ${SDK}/usr/lib/swift-5.0/iphoneos/libswiftsimd.dylib -r -o ${DST}/libswiftsimd.dylib
```
Comment (JCash) [inline facebook/libhelper/ios/update.sh:42]: Using the copy step that XCode uses when building a test app using the same Facebook libraries as this extension.

File: game.project [hunk 3]
```diff
@@ -1,6 +1,7 @@
 [project]
 dependencies = https://github.com/andsve/dirtylarry/archive/master.zip,
 title = extension-facebook
+bundle_resources = /facebook/bundle_res
```
Comment (JCash) [inline game.project:4]: We don't have a better way of doing this (yet)

**Review by JCash — COMMENTED** [review]
This change requires two things from the main defold repo:
* Improved support for the "-rpath" flag (currently doesn't allow the `@` sign)
* Signing of the files in the `./Frameworks` path

File: game.project (hunk 3 above)
Comment (britzl) [reply game.project:4]: Does this mean our users need to do the same?

Comment (britzl) [reply game.project:4]: Can't we put them in facebook/res/ios?

Comment (JCash) [reply game.project]: Yes, I had just forgot about it :)

**Review by britzl — APPROVED** [review]

**Pull request merged**